samples/backups-create.js

```diff
@@ -47,12 +46,17 @@ async function createBackup(instanceId, databaseId, backupId, projectId) {
     const databasePath = database.formattedName_;
     // Expire backup 14 days in the future
     const expireTime = Date.now() + 1000 * 60 * 60 * 24 * 14;
-    // Create a backup of the state of the database at the earliest possible
-    // version time.
+    // Version time is the server's current time
+    const query = {
```

Comment:
Same as for the Go sample: Would it be better to move this out of the sample code?

Also, you need to [change this back](https://github.com/olavloite/nodejs-spanner/blob/84045a145e4dfabb4d5c5fc1b0393dff11aae92a/samples/system-test/spanner.test.js#L852) to 7.
(See the changes in https://github.com/googleapis/nodejs-spanner/pull/1310/files)

Reply:
Agree with @olavloite as for the Go sample.

Reply:
Parameterised the version time in the samples